nncf/experimental/openvino_native/graph/metatypes/openvino_metatypes.py

```diff
@@ -576,6 +576,12 @@ class OVClampMetatype(OVOpMetatype):
     op_names = ['Clamp']
 
 
+@OV_OPERATOR_METATYPES.register()
+class OVSquaredDifferenceMetatype(OVOpMetatype):
```

Comment:
Shi=old we add `OVSquaredDifferenceMetatype` to `QUANTIZABLE_OPERATIONS`?

Reply:
I think - yes. Added